ci-operator/config/maistra/istio/maistra-istio-maistra-3.0.yaml

```
@@ -96,6 +96,55 @@ tests:
           memory: 100Mi
       timeout: 20m0s
     workflow: servicemesh-istio-e2e-aws
+- as: maistra-istio-unit-3-0
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h0m0s
+    version: "4.10"
+  steps:
+    env:
+      BUILD_WITH_CONTAINER: "0"
+      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:upstream-master
+      MAISTRA_NAMESPACE: maistra-e2e-test
+      MAISTRA_SC_POD: maistra-e2e-test-sc-pod
+    test:
+    - as: copy-src
+      cli: latest
+      commands: |
+        # SRC_PATH does end with /. : the content of the source directory is copied into dest directory
+        oc cp ./. "${MAISTRA_NAMESPACE}"/"${MAISTRA_SC_POD}":/work/
+      env:
+      - name: MAISTRA_NAMESPACE
+      - name: MAISTRA_SC_POD
+      from: src-test
+      resources:
+        requests:
+          cpu: 100m
+          memory: 100Mi
+      timeout: 20m0s
+    - as: maistra-istio-unit-3-0
+      cli: latest
+      commands: |
+        oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" \
+          bash -c \
+          "cd /work;
+          entrypoint \
+          make -e "T=-v -count=1" build racetest binaries-test"
+        oc cp "${MAISTRA_NAMESPACE}"/"${MAISTRA_SC_POD}":"${ARTIFACT_DIR}"/. "${ARTIFACT_DIR}"
+      env:
+      - name: BUILD_WITH_CONTAINER
+      - name: MAISTRA_NAMESPACE
+      - name: MAISTRA_SC_POD
+      from: src-test
+      resources:
+        requests:
+          cpu: "2"
+          memory: 2Gi
```

```suggestion
          cpu: 100m
          memory: 100Mi
```
This section only runs `oc rsh`  100m resource is enough. The real test execution runs in the remote pod and resources are defined as the node type  COMPUTE_NODE_TYPE: m5d.4xlarge above.